spec/post_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  context 'test posts' do
+    before :each do
+      @user = User.create(name: 'william', photo: 'https://github.com/kidde60', bio: 'web developer')
+      @post = Post.create(author: @user, title: 'Title', text: 'My sample text')
+    end
+
+    it 'should not have an empty title' do
+      expect(@post.title).to eql 'Title'
+    end
+
```

Comment:
- [ ] Kindly add tests for post validations (Title must not be blank, CommentsCounter must be an integer greater than or equal to zero, LikesCounter must be an integer greater than or equal to zero) as you did in the user model tests

Reply:
Thanks @gemmen29 for the comment but have included the tests for the post validation.